pycaret/nlp/__init__.py

```diff
@@ -14,7 +14,12 @@
 from pycaret.loggers.base_logger import BaseLogger
 from pycaret.loggers.dagshub_logger import DagshubLogger
 from pycaret.loggers.mlflow_logger import MlflowLogger
+<<<<<<< HEAD
```

Comment:
Looks like there is a conflict. Can you fix this?

Reply:
should be fixed now!